class.csstidy_optimise.php

```diff
@@ -134,6 +134,13 @@ public function postparse() {
 				}
 			}
 		}
+
+		// Custom fix for badly formatted comments that look like this: /*!* RESET STYLES *!*/
```

Comment:
I'm now understanding that your goal is to remove this comments like `/*!* RESET STYLES *!*/` from the output.
But this is a feature that allows comments starting by `/*!` to be kept in the final output to be able to keep authoring or licensing mentions in compressed CSS

see https://github.com/Cerdic/CSSTidy/commit/a834d4d2

This is a convention introduced by saas and we are folllowing in CSSTidy
See https://sass-lang.com/documentation/syntax/comments/
"[comments] can start with /*! to avoid being stripped in compressed mode."